mod_modular_vanilla/hooks/ai/tactical/behavior.nut

```diff
@@ -0,0 +1,10 @@
+::ModularVanilla.QueueBucket.VeryLate.push(function() {
+	::ModularVanilla.MH.hook("scripts/ai/tactical/behavior", function(q) {
```

Comment:
Maybe it is worth considering to make this a `hookTree` instead of `hook`. In Reforged I have a case where I overwrite the `function queryTargetValue` inside `ai_rf_attack_lunge` behavior. Although that overwrite still calls the base function but it does so at the start of the function so the multiplier returned by this skills event won't apply to the full value returned by that function. It is also possible that someone overwrites this function completely for certain behaviors and that will cause our events to not fire for that behavior.